tests/integration/test_syscollector/test_syscollector_configuration.py

```diff
@@ -0,0 +1,457 @@
+    non_critical_fields = ('max_eps')
+    file_monitor = FileMonitor(LOG_FILE_PATH) if sys.platform == 'win32' else None
+
+    if field is not None:
+        if field == 'hotfixes' and sys.platform != 'win32':
+            return True
+
+        evm.check_tag_error(file_monitor=file_monitor, field=field, timeout=T_10)
+
+        if field in non_critical_fields:
+            # Check that the module has started if the field is not critical
+            file_monitor = FileMonitor(LOG_FILE_PATH)
+            evm.check_has_started(file_monitor=file_monitor, timeout=T_10)
+            return True
+    else:
+        evm.check_attr_error(file_monitor=file_monitor, attr=attribute, timeout=T_10)
+
+    # Check that the module does not start
+    with pytest.raises(TimeoutError):
+        evm.check_has_started(file_monitor=file_monitor, timeout=T_10)
+        pytest.fail(f"The module has started anyway. This behaviour is not the expected.")
+
+
+@pytest.mark.parametrize('configuration, metadata', zip(t4_configurations, t4_config_metadata), ids=t4_case_ids)
+@pytest.mark.xfail(reason='Reported in wazuh/wazuh#15413')
+def test_syscollector_default_values(configuration, metadata, set_wazuh_configuration,
+                                     configure_local_internal_options_module, truncate_log_file,
+                                     daemons_handler_function):
+    '''
+    description: Check that the module sets the default values when the configuration block is empty.
+
+    test_phases:
+        - Configure syscollector.
+        - Configure modulesd in debug mode.
+        - Truncate the log.
+        - Restart analysisd, wazuh-db and modulesd
+
+    wazuh_min_version: 4.4.0
+
+    tier: 0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration loaded from the template located in `configuration` folder.
+        - metadata:
+            type: dict
+            brief: Test case metadata.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set wazuh configuration using the configuration template.
+        - configure_local_internal_options_module:
+            type: fixture
+            brief: Configure the local internal options file.
+        - truncate_log_file:
+            type: fixture
+            brief: Truncate the log file before and after the test execution.
+        - daemons_handler_function:
+            type: fixture
+            brief: Handler of Wazuh daemons for each test case.
+
+    assertions:
+        - Check if the module sets the default configuration.
+
+    input_description:
+        - The `configuration_syscollector_no_tags.yaml` file provides the module configuration for this test.
+        - The `case_test_default_values.yaml` file provides the test cases.
+    '''
+    file_monitor = FileMonitor(LOG_FILE_PATH) if sys.platform == 'win32' else None
+    evm.check_config(file_monitor=file_monitor, timeout=T_10)
```

Comment:
In addition to validating that the set configuration is correct, I would like you to also validate that the first start syscollector logs appear.

Reply:
Done in f79e919